finalitygadget/finalitygadget.go

```diff
@@ -11,9 +11,9 @@ import (
 	"sync"
 	"time"
 
-	bbnClient "github.com/babylonlabs-io/babylon/client/client"
+	bbnclient "github.com/babylonlabs-io/babylon/client/client"
 	bbncfg "github.com/babylonlabs-io/babylon/client/config"
-	"github.com/babylonlabs-io/finality-gadget/bbnclient"
+	fgbbnclient "github.com/babylonlabs-io/finality-gadget/bbnclient"
```

Comment:
how's this diff from bbnclient?

Reply:
We have `bbnclient`, and before changing the package, it is `bbnClient` but we have the variable named `bbnClient` in the code, it would be shadow.
```
bbnclient "github.com/babylonlabs-io/babylon/client/client"
```